apps/sv/src/main/scala/org/lfdecentralizedtrust/splice/sv/onboarding/joining/JoiningNodeInitializer.scala

```diff
@@ -168,7 +162,10 @@ class JoiningNodeInitializer(
           )
         ),
       ).tupled
-      decentralizedSynchronizerId <- connectToDomainUnlessMigratingDsoParty(dsoPartyId)
+      // It is possible that the participant left disconnected to domains due to party migration failure in the last SV startup.
```

Comment:
this comment is a bit weird, if party migration failed in the startup then in most scenarios reconnecting is exactly what you don't want if you haven't imported the acs yet